seqr/views/apis/staff_api_tests.py

```diff
@@ -195,3 +202,75 @@ def test_elasticsearch_status(self, mock_elasticsearch, mock_get_mapping):
                                                       h="index,docs.count,store.size,creation.date.string")
         mock_es_client.cat.aliases.assert_called_with(format="json", h="alias,index")
         mock_get_mapping.assert_called_with(doc_type='variant,structural_variant')
+
+    def test_mme_details(self):
+        url = reverse(mme_details)
+        _check_login(self, url)
+
+        response = self.client.get(url)
+        self.assertEqual(response.status_code, 200)
+        response_json = response.json()
+        self.assertListEqual(response_json.keys(), ['metrics', 'genesById', 'submissions'])
+
+    def test_seqr_stats(self):
+        url = reverse(seqr_stats)
+        _check_login(self, url)
+
+        response = self.client.get(url)
+        self.assertEqual(response.status_code, 200)
+        response_json = response.json()
+        self.assertListEqual(response_json.keys(), ['individualCount', 'familyCount', 'sampleCountByType'])
+
+    def test_get_projects_for_category(self):
+        url = reverse(get_projects_for_category, args=[PROJECT_CATEGRORY_NAME])
+        _check_login(self, url)
+
+        response = self.client.get(url)
+        self.assertEqual(response.status_code, 200)
+        response_json = response.json()
+        self.assertListEqual(response_json.keys(), ['projectGuids'])
+
+    def test_discovery_sheet(self):
+        non_project_url = reverse(discovery_sheet, args=[NON_PROJECT_GUID])
+        _check_login(self, non_project_url)
+
+        response = self.client.get(non_project_url)
+        self.assertEqual(response.status_code, 500)
+
+        empty_project_url = reverse(discovery_sheet, args=[PROJECT_EMPTY_GUID])
+
+        response = self.client.get(empty_project_url)
+        self.assertEqual(response.status_code, 200)
+        response_json = response.json()
+        self.assertListEqual(response_json.keys(), ['rows', 'errors'])
+        self.assertListEqual(response_json['rows'], [])
+        self.assertListEqual(response_json['errors'], ["No data loaded for project: Empty Project"])
+
+        url = reverse(discovery_sheet, args=[PROJECT_GUID])
+        response = self.client.get(url)
+        self.assertEqual(response.status_code, 200)
+        response_json = response.json()
+        self.assertListEqual(response_json.keys(), ['rows', 'errors'])
+        self.assertListEqual(response_json['errors'], [u'No data loaded for family: 13. Skipping...'])
```

Comment:
Please test the content of the rows

Reply:
Can I just test the length of the content list of the rows? Because I don't know what these contents from and will they change over time.

Reply:
This is a crucial report and it is incredibly important that the content does not change. Please test at least one of the row's content.

Reply:
Added testing one of the rows.